nodejs/eks/cluster.ts

```diff
@@ -435,6 +500,11 @@ export interface ClusterOptions {
      */
     serviceRole?: pulumi.Input<aws.iam.Role>;
 
+    /**
+     * The IAM Role used to create & authenticate against the EKS cluster.
```

Comment:
Let's add a note about how this role is granted `system:masters` permissions in k8s, and possibly link to https://docs.aws.amazon.com/eks/latest/userguide/add-user-role.html